website/docs/getting-started/setup/installation-guides/aws-ami.md

```diff
@@ -24,7 +24,9 @@ Follow these steps to install Appsmith using an Amazon Machine Image (AMI):
 4. On the **Launch instance** screen, scroll down to the **Application and OS images(Amazon Machine Image)** section.
 5. Search for **Appsmith** in the search bar.
 6. In the search results, click **AWS Marketplace AMIs**.
-7. Select the **Appsmith** image.
+7. Select the image:
+    * **Appsmith** for **Community Edition**.
```

Comment:
```suggestion
    * **Appsmith Community Edition** for **Community Edition**.
```

Reply:
There still appears to be something awkward here. Can you share a screenshot of the "Select the image" option on the UI or however it appears. From the text, I'm not able to visualise how this would be.

Reply:
This is how it looks. It's fairly simple, I don't think we need to add an image.
![aws-ami-appsmith-images](https://github.com/appsmithorg/appsmith-docs/assets/99252011/48421885-5e5b-4723-a314-2636505766bf)

Reply:
No it was not for adding in documentation. I wanted to see to understand if we can clarify that line to something like:

> Select the image from the two options based on whether you want to install the Community or Business Edition.
    - Appsmith Community Edition
    - Appsmith Business Edition

Reply:
rewritten it.